x/gov/handler_test.go

```diff
@@ -1218,7 +1183,8 @@ func TestUpdateElectorate(t *testing.T) {
 				Title:                 "fooo",
 				Electors:              []Elector{{Address: bobby, Weight: 10}},
 				TotalElectorateWeight: 10,
-				UpdateElectionRuleID:  weavetest.SequenceID(1),
+				UpdateElectionRuleRef: orm.VersionedIDRef{ID: weavetest.SequenceID(1), Version: 1},
```

Comment:
Good to add this.
I would be happy to see a case that was not just 1->2. Maybe even 2->3 :) Or an attempt at 1->3 failing

Reply:
1->3 failing can not be tested with the update msg as it modifies the latest version. But there is a test in the orm package to prevent gaps: https://github.com/iov-one/weave/blob/master/orm/versioning_bucket_test.go#L99

What do you expect to learn from 2->3? Can this go into orm package tests instead?